src/components/MobileWalletDialog/utils.ts

```diff
@@ -0,0 +1,58 @@
+import type { Keystore } from '@shapeshiftoss/hdwallet-native-vault'
+import { decoder, encoder } from '@shapeshiftoss/hdwallet-native-vault/dist/util'
```

Comment:
suggestion: this is `TextDecoder` which is available natively for 10+ years, not sure why hdwallet polyfills it using `web-encoding`  🤡 but we should just be able to use `TextDecoder` ootb here without  hdwallet dependencies (same goes for `TextEncoder`)
<img width="503" alt="image" src="https://github.com/user-attachments/assets/58ec6582-3c7d-4810-8c77-b189ada5913f" />

Reply:
1608d033a7